src/format/packed/mod.rs

```diff
@@ -51,18 +42,72 @@ pub enum Variant {
 #[cfg(debug_assertions)]
 impl<T: Element> ::format::Validate for Packed<T> {
     fn validate(&self) {
-        assert_eq!(self.values.len(), storage!(self.size));
+        assert_eq!(self.values.len(), triangular_elems(self.size));
     }
 }
 
 size!(Packed, size, size);
 
+/// Computes the offset into the array for the upper triangular representation
```

Comment:
Change to ”Compute the offset into an array with the upper triangular representation.” (Phrase as a command and finish with a period.)